app/util/IpmiCommand.scala

```diff
@@ -72,7 +72,7 @@ abstract class IpmiCommand extends BackgroundProcess[Option[CommandResult]] {
     Some(cr)
   }
 
-  override protected def defaultTimeout: Duration = Duration(2, TimeUnit.MILLISECONDS)
```

Comment:
This should use the timeout in concurrency_reference.conf. It was overridden, we should remove it as part of some later PR. We may also want to break out the timeout for ipmi from the timeouts for other background operations at the time.